pathod/utils.py

```diff
@@ -69,7 +71,10 @@ def escape_unprintables(s):
     """
     s = s.replace("\r\n", "PATHOD_MARKER_RN")
     s = s.replace("\n", "PATHOD_MARKER_N")
-    s = inner_repr(s)
+    if six.PY2:
+        s = inner_repr(s)
+    else:
+        s = s.encode('unicode_escape').decode('ascii')
```

Comment:
`escape_unprintables` is used [here](https://github.com/dufferzafar/mitmproxy/blob/pathod-port/pathod/app.py#L169).

The input to it is bytes in Py2, and unicode in Py3.